README.md

```diff
@@ -104,6 +104,12 @@ Make sure to define `COVERITY_SCAN_TOKEN` in your
     # Default: 'make'
     command: ''
 
+    # Specify the working-directory of where to run commands.
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun
+    #
```

Comment:
prob worth documenting that the directory must exist beforehand as it is not created automatically

Reply:
Fixed.